sdk/core/azure-core/CHANGELOG.md

```diff
@@ -15,6 +15,14 @@
 
 - Added `TokenRefreshOptions()` to `TokenCredential`, with a default token refresh offset of 2 minutes, and a default token refresh retry timeout of 30 seconds.
 
+## 1.7.0-beta.2 (2020-07-23)
```

Comment:
Looks like we already have this on L10 above

Reply:
Removed